src/herder/HerderSCPDriver.cpp

```diff
@@ -692,6 +703,9 @@ HerderSCPDriver::combineCandidates(uint64_t slotIndex,
 
     // take the txSet with the biggest size, highest xored hash that we have
     TxSetFramePtr bestTxSet;
+    TimePoint bestTxSetCloseTime = 0;
+    LedgerCloseValueSignature highestTxSetSignature;
+    auto const newCloseTimeBehavior = curProtocolContainsClosetimeChange();
```

Comment:
Nitpick: same thing as for the global version constant. This name doesn't tell me anything about what that behavior is, or even what "new" means. I can understand it just fine in the context of this PR, but when someone maintains this code in a year or two this name won't mean anything to them.

Reply:
I've changed it to `curProtocolPreservesTxSetCloseTimeAffinity()`, matching the CAP title (and thus the new constant name as well).